packages/datadog-plugin-google-cloud-vertexai/src/tracing.js

```diff
@@ -33,7 +33,10 @@ class GoogleCloudVertexAITracingPlugin extends TracingPlugin {
       }
     }, false)
 
-    const tags = this.tagRequest(request, instance, stream, span)
+    const model = extractModel(instance)
+    const tags = {
+      'vertexai.request.model': model
+    }
```

Comment:
just one more cleanup nit i promise 😅 we can bring this up to the `meta` field in the `this.startSpan` call by doing

```javascript
meta: {
  [MEASURED]: 1,
  'vertexai.request.model': extractModel(instance)
}
```

instead of doing it separately